Updated Readme

piratekingdev opened the pull request:
I modified the Readme to contain useful information about the project. This should help streamline the process of explaining what Dave is all about and answering commonly asked questions people might have.

Dizeeee commented:
I just updated the readme to resolve some of these issues, could you add some of the info on here that I missed to the readme?

piratekingdev commented:
> I just updated the readme to resolve some of these issues, could you add some of the info on here that I missed to the readme?

I added the relevant information to the new Readme I am purposing and resolved the conflict.

brussell98 commented on README.md:
```diff
@@ -1,11 +1,15 @@
-# dave
+# Description
+Dave automates virtually everything is a general-purpose Discord bot designed to help server operators administrate and engage with their community.
```
This is false. Dave does not automate anything in its current state. (Also grammar)

Dizeeee replied:
I would prefer dave to be kept lowercase as part of the branding

piratekingdev replied:
> I would prefer dave to be kept lowercase as part of the branding

I modified the readme in the commit to make all reference to dave lowercase and removed fluff text that did not add much value.

piratekingdev commented:
Just updating this PR to let you know I added the initial prettier configuration to help close that issue.

Dizeeee commented on .prettierrc:
```diff
@@ -0,0 +1,11 @@
+{
```
Can you remove values set here that are already defaults? [List of options and defaults here, just ctrl+f through it.](https://prettier.io/docs/en/options.html)
